crates/bevy_pbr/src/pbr_material.rs

```diff
@@ -105,15 +139,43 @@ pub struct StandardMaterial {
 
     /// Support two-sided lighting by automatically flipping the normals for "back" faces
     /// within the PBR lighting shader.
-    /// Defaults to false.
+    ///
+    /// Defaults to `false`.
     /// This does not automatically configure backface culling, which can be done via
     /// `cull_mode`.
     pub double_sided: bool,
-    /// Whether to cull the "front", "back" or neither side of a mesh
-    /// defaults to `Face::Back`
+
+    /// Whether to cull the "front", "back" or neither side of a mesh.
+    ///
+    /// Defaults to `Face::Back`.
```

Comment:
```suggestion
    /// Defaults to `Face::Back` if set to `None`.
```
?

Reply:
A quick glance at the source tells me this sets the `cull_mode` field of [`PrimitiveState`](https://docs.rs/wgpu/latest/wgpu/struct.PrimitiveState.html#structfield.cull_mode) Which links to <https://gpuweb.github.io/gpuweb/#enumdef-gpucullmode>. Which indicates that `None` is its own individual flag, separate from `Back` and `Front`. I'll clarify the documentation and explain what `None` does though.

Reply:
I see that the default for `StandardMaterial` is indeed `Some(Face::Back)`. I understand your remark, and you're right, but I got confused originally. Maybe adding the `Some` would clarify things:

```
/// Defaults to `Some(Face::Back)`.
```

Reply:
Yep, that's what I did, that's indeed confusing. I also clarified the first sentence to make explicit that "otherwise" means `None`.